From cc7746c141d976a4779cf868774fae1fe7627a6d Mon Sep 17 00:00:00 2001
From: Silvio Fricke <silvio.fricke@gmail.com>
Date: Thu, 13 Jan 2022 16:06:58 +0100
Subject: [PATCH] build: fix i18n usage on data/meson.build for meson 0.61.0

With the latest meson the code doesn't compile anymore and reports an
error:

	meld/data/meson.build:34:0: ERROR: Function does not take positional arguments.

This patch tries to fix that problem.

Close #645

Signed-off-by: Silvio Fricke <silvio.fricke@gmail.com>
---
 data/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/meson.build b/data/meson.build
index 01949188..de099d3b 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -32,7 +32,6 @@ if desktop_file_validate.found()
 endif
 
 appdata_file = i18n.merge_file(
-  'appdata',
   input: configure_file(
     input: files('org.gnome.meld.appdata.xml.in.in'),
     output: 'org.gnome.meld.appdata.xml.in',
-- 
GitLab

